dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:14-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Copy the package.json and package-lock.json files to the working directory
+COPY package*.json ./
```

Comment:
The repository contains a yarn.lock file, so it should grab that file and install with `yarn` instead of `npm install` to grab the frozen dependencies.

Reply:
Yep, lets make this change 👍